### DISCUSSION TITLE
what's the difference between the hash ending of files in .pnpm ?

### DISCUSSION THREAD

**Discussion by KyrieLii** [pnpm/pnpm#5383] [Q&A] [answered]
in pnpm workspace `root_path/node_modules/.pnpm` , i want to know what's the difference with this files , thanks !

<img width="661" alt="image" src="https://user-images.githubusercontent.com/16858738/191462839-05c8c838-ce72-4b41-bb8b-a02847b174e8.png">

**Comment by zkochan** [comment, marked as answer]
It is a hash of the resolved peer dependencies (https://pnpm.io/how-peers-are-resolved)

For instance, in one case it is a hash from `foo@1.0.0` and in another case it is a hash from `foo@1.2.0`

**Reply by KyrieLii** [reply]
Thanks for your answer !
So can i understand it as a deeper peer dependancies identifier , because the string will be too long if  only use file name and version number ?
